chips/stm32f401cc/src/interrupt_service.rs

```diff
@@ -29,6 +29,7 @@ impl<'a> Stm32f401ccDefaultPeripherals<'a> {
 }
 impl<'a> kernel::platform::chip::InterruptService for Stm32f401ccDefaultPeripherals<'a> {
     unsafe fn service_interrupt(&self, interrupt: u32) -> bool {
+        #[allow(clippy::match_single_binding)]
```

Comment:
I guess this is OK because eventually someone might refine it more than just calling the other `service_interrupt` unconditionally?

Reply:
Yeah, I didn't want to just remove the `// put Stm32f401cc specific interrupts here` comment which seemed helpful.